packages/node_modules/@webex/widget-meet/src/components/call-active/index.js

```diff
@@ -10,7 +10,7 @@ import {
 
 import styles from './styles.css';
 
-const ActiveMeeting = ({adapter, meetingId}) => {
+const ActiveMeeting = ({setEscapedAuthentication,adapter, meetingId}) => {
```

Comment:
When a user escapes the authentication modal we set escapesAuth state as true. This allows close the meeting UI.